docs/admin/upgrade.md

```diff
@@ -6,10 +6,4 @@ Here you will find any steps necessary to upgrade the App in your Nautobot envir
 
 When a new release comes out it may be necessary to run a migration of the database to account for any changes in the data models used by this app. Execute the command `nautobot-server post-upgrade` within the runtime environment of your Nautobot installation after updating the `nautobot-golden-config` package via `pip`.
 
-## v1.0.0
-
```

Comment:
Looks like this file was replaced, please revert

Reply:
Fixed by cleanup